custom_components/ha_kia_hyundai/manifest.json

```diff
@@ -7,6 +7,6 @@
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/dahlb/ha_kia_hyundai/issues",
   "loggers": ["ha_kia_hyundai", "kia_hyundai_api"],
-  "requirements": ["kia-hyundai-api==1.4.3"],
-  "version": "1.12.2"
```

Comment:
don't change the version string, it is managed by github workflows based on commit message (this being a feat: pr will auto increment it to 1.13.0)